### PR TITLE
Add openshift/cluster-api-operator CI

### PR DESCRIPTION
**Pull request by Fedosin**
This is a new [component](https://github.com/openshift/cluster-api-operator) forked from upstream https://github.com/kubernetes-sigs/cluster-api-operator

### CODE REVIEW

File: core-services/prow/02_config/openshift/cluster-api-operator/_pluginconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - openshift/cluster-api-operator
+  require_self_approval: false
+lgtm:
+- repos:
+  - openshift/cluster-api-operator
+  review_acts_as_lgtm: true
```
Comment (alexander-demichev) [inline core-services/prow/02_config/openshift/cluster-api-operator/_pluginconfig.yaml:9]: should this be false?

Comment (Fedosin) [reply core-services/prow/02_config/openshift/cluster-api-operator/_pluginconfig.yaml:9]: That was set by default, let's remove it.

File: core-services/image-mirroring/openshift/mapping_origin_4_11 [hunk 2]
```diff
@@ -43,6 +43,7 @@ registry.ci.openshift.org/origin/4.11:cloud-credential-operator quay.io/openshif
 registry.ci.openshift.org/origin/4.11:cloud-event-proxy quay.io/openshift/origin-cloud-event-proxy:4.11 quay.io/openshift/origin-cloud-event-proxy:4.11.0 quay.io/openshift/origin-cloud-event-proxy:latest
 registry.ci.openshift.org/origin/4.11:cloud-network-config-controller quay.io/openshift/origin-cloud-network-config-controller:4.11 quay.io/openshift/origin-cloud-network-config-controller:4.11.0 quay.io/openshift/origin-cloud-network-config-controller:latest
 registry.ci.openshift.org/origin/4.11:cluster-api-actuator-pkg quay.io/openshift/origin-cluster-api-actuator-pkg:4.11 quay.io/openshift/origin-cluster-api-actuator-pkg:4.11.0 quay.io/openshift/origin-cluster-api-actuator-pkg:latest
+registry.ci.openshift.org/origin/4.11:cluster-api-operator quay.io/openshift/origin-cluster-api-operator:4.11 quay.io/openshift/origin-cluster-api-operator:4.11.0 quay.io/openshift/origin-cluster-api-operator:latest
```
Comment (bparees) [inline core-services/image-mirroring/openshift/mapping_origin_4_11:46]: based on my thinking in https://coreos.slack.com/archives/CBZHF4DHC/p1644597800613349?thread_ts=1644593799.371139&cid=CBZHF4DHC

and the guidlines i am trying to describe in https://github.com/openshift/openshift-docs/pull/41803

I think this should be `cluster-kube-cluster-api-operator` so long as it is going to be a CVO operator and/or can be considered to impact the config/behavior of the cluster it is running on (which cluster-api does/can through machine management, even if today we do not use it in that way)

**Comment by bparees** [comment]
/approve

image name is approved

File: core-services/prow/02_config/_plugins.yaml [hunk 3]
```diff
@@ -256,27 +256,6 @@ cherry_pick_unapproved:
   comment: This PR is not for the master branch but does not have the `cherry-pick-approved`  label.
     Adding the `do-not-merge/cherry-pick-not-approved`  label.
 config_updater:
-  cluster_groups:
```
Comment (droslean) [inline core-services/prow/02_config/_plugins.yaml:259]: why this file has been changed? /hold

Comment (petr-muller) [reply core-services/prow/02_config/_plugins.yaml:259]: It was a bug in repo-init (DPTP-2721) that should be fixed now but the PR was likely prepared by a version before the fix

Comment (petr-muller) [reply core-services/prow/02_config/_plugins.yaml:259]: The change in this file needs to be removed from the PR @Fedosin

Comment (Fedosin) [reply core-services/prow/02_config/_plugins.yaml]: @petr-muller @droslean thanks! I removed that generated piece of code.

**Review by petr-muller — APPROVED** [review]

**Pull request merged**